### PR TITLE
[SDPT-690] Add technologies to all existing posts

### PR DESCRIPTION
**Pull request by youminhan**
Besides adding technolgoies to all existing posts, this PR also updates the tags to keep consistency. 

Signed-off-by: Youmin Han <Youmin.Han@dell.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,9 +71,9 @@ some excerpt
 
 + **layout**: Set to ***post***.
 + **category**: Choose one of the following categories: ``Getting Started``, ``Ingesting Data``, ``Processing Data``, ``Data Output``, ``Demos``, ``Connectors``.
-+ **tags**: Put as many as are relevant. Tags will be shown on the Tags page.
++ **tags**: Put as many as are relevant. Tags will be shown on the Tags page. All tags must be lowercase.
 + **subtitle**: This will show beneath the title on the post details page.
-+ **technologies**: List what technologies(less or equal than 2) is used in this post. This will show on the right panel.
++ **technologies**: List what technologies(less or equal than 2) is used in this post. This will show on the right panel. The first letter of each technology should be capitalized.
```
Comment (twincitiesguy) [inline README.md]: can we reword this to say "Technologies listed here should be appropriately capitalized." -- we don't want to force just the first letter; i.e. if it's an acronym, like SDP, or something like eTag

**Review by twincitiesguy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (youminhan) [reply README.md]: That makes sense. I just make the changes to the README.

**Review by twincitiesguy — APPROVED** [review]
